packages/internal/src/build/api.ts

```diff
@@ -50,8 +50,16 @@ export const prebuildApiFiles = (srcFiles: string[]) => {
       return undefined
     }
 
+    // Fixes nodeFileTrace errors on windows when building
+    let nodeFileTraceCompliantCode: string = result.code
+    if (process.platform === 'win32') {
```

Comment:
Since we don't run NFT in the regular build process, would it be possible to only make the change in the server less deploy step? It's in a file called nftPack.ts I think! 

It would be great also if we could specifically target import paths and use "ensurePosixPath", because the replaceAll here may have unintended side effects!

Reply:
I also thought of `ensurePosixPath`, but it's not exactly the same. 
`ensurePosixPath` works on paths like these: `C:\Users\Tobbe\dev\my-rw-project`
The code here seems to expect paths that use double backslashes `\\`.
So using `ensurePosixPath` you might end up with `/c//Users//Tobbe//dev//my-rw-project`. That's why I didn't recommend using that function.

But I haven't actually tested the code, so of course I could be wrong 🙂

Reply:
I `console.loged` every single thing in every file related to packing and zipping.
I did test all the posix pathing, even forced the entire `paths.ts` file to output only posix pathing. but babel just wanted to windows it up anyways. like the right pathing went in, but then the `results.code` output `..\\..\\\whatever\\this\\pathing\\is `
and `nodeFileTrace` couldn't read the ` \\path\\to\\something\\db`. I do not know enough about the inner workings of babel or nodeFileTrace, but maybe its a fix that could be applied to `nodeFileTrace` instead of our workaround. I'll poke around in `nodeFileTrace` I saw some issues related to windows in their discussions.

Reply:
Ah interesting.... so the issue is that nft doesn't seem to like the imports that are done by babel on windows (as you've suggested).

Couple questions since you've debugged this thoroughly (and I forget things quickly):
a) Are you noticing the problem happening with all imports or just `src/lib/*` imports?
b) Any chance you could upload a gist/zip with the output in `.redwood/prebuild` and `api/dist` so I could take a closer look? I can do this but it might help to work from the same codebase. 

My gut feeling is saying its to do with one of the babel plugins we use - either `packages/internal/src/build/babelPlugins/babel-plugin-redwood-src-alias` or `packages/internal/src/build/babelPlugins/babel-plugin-redwood-directory-named-import.ts`

Reply:
It is only hapenning with the lib imports
 ``` 
import { getCurrentUser } from "..\\lib\\auth";
import { db } from "..\\lib\\db";
import { logger } from "..\\lib\\logger"; 
```

Reply:
So I think..... the fix might be in the `packages/internal/src/build/babelPlugins/babel-plugin-redwood-src-alias` plugin then! This plugin's job is to take imports like `src/lib/auth` and convert them. 

In the process I think... what must be happening is that on windows its adding these double slashes. 

So the first thing I would do is create a unit test (see [this](https://github.com/redwoodjs/redwood/blob/main/packages/internal/src/build/babelPlugins/__tests__/babel-plugin-redwood-directory-named-imports.ts) for reference, where you need to add "inputs" and "outputs" to the __fixtures__ folder. It sounds complicated but really isn't :) 

Thank you so much for your help with this, and please do let me know if this feels to complex/annoying to you I can take over when I have a chance